src/ducks/myApps/Container.jsx

```diff
@@ -12,7 +12,10 @@ const mapStateToProps = (state, ownProps) => ({
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
   fetchApps: () => dispatch(fetchApps()),
-  uninstallApp: (appSlug) => dispatch(uninstallApp(appSlug))
+  uninstallApp: (appSlug) => {
+    return dispatch(uninstallApp(appSlug))
+    .then(() => dispatch(fetchApps()))
```

Comment:
Do we really need to fetch ? `uninstallApp` should remove the app from the state, shouldn't it ?